deepface/detectors/FastMtCnn.py

```diff
@@ -2,6 +2,7 @@
 import cv2
 import numpy as np
 from deepface.models.Detector import Detector, FacialAreaRegion
+import torch
```

Comment:
this is not a must dependency in deepface. users will have import error if this import is put in global. right?